queries/solidity/highlights.scm

```diff
@@ -89,34 +89,38 @@
 
 ; Keywords
 [
- "pragma"
  "contract"
  "interface"
  "library"
  "is"
  "struct"
  "enum"
  "event"
- "using"
  "assembly"
  "emit"
+ "modifier"
+ "var"
+ (virtual)
+ (override_specifier)
+ (yul_leave)
```

Comment:
It might be more appropriate for `(yul_leave)` to be parked under `@keyword.return`, as it is used to return directly from the equivalent of inline assembly.